pydis_site/templates/resources/resources.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'base/base.html' %}
+{% load static %}
+
+{% block title %}Resources{% endblock %}
+{% block head %}
+  <link rel="stylesheet" href="{% static "css/resources/resources.css" %}">
+{% endblock %}
+
+{% block content %}
+  {% include "base/navbar.html" %}
+
+  <section class="breadcrumb-section section">
+      <div class="container">
+          <nav class="breadcrumb is-pulled-left" aria-label="breadcrumbs">
+              <ul>
+                  <li class="is-active"><a href="/resources">Resources</a></li>
+              </ul>
+          </nav>
+      </div>
+  </section>
```

Comment:
Including breadcrumbs at the index feels redundant, especially since there's the "Resources" title right below. Does anyone else feel the same way? Is it better to keep it for consistency?

Reply:
As other resources pages will have it too, I think it is better to leave it.

Reply:
The previous version removed/hid it because it was considered redundant and we'd have a page title, a breadcrumb and a content title all with the same word, which looked kinda weird. Take that how you will.

Reply:
Actually I removed this breadcrumb now.